DATA/Par_file

```diff
@@ -1,8 +1,8 @@
-#-----------------------------------------------------------
+#-----------------------------------------------------------------------------
 #
-# Simulation input parameters
+# simulation input parameters
```

Comment:
the current Par_file format uses capital "Simulation" and has some more verbose comments (see for example after `ATTENUATION_f0_REFERENCE`) -> please base the change of all Par_files from a current Par_file and only add the two new parameters for permittivity and conductivity.